spec/system/eligibility_spec.rb

```diff
@@ -275,13 +275,9 @@ def then_i_do_not_see_the_start_page
   end
 
   def then_i_see_the_start_page
-    expect(page).to have_content(
-      "Check your eligilibity to apply for qualified teacher status (QTS) in England"
-    )
-    expect(page).to have_current_path("/eligibility/start")
-    expect(page).to have_title(
-      "Check your eligilibity to apply for qualified teacher status (QTS) in England"
-    )
+    expect start_page.heading.text.to eq(
```

Comment:
This is still a bit wonky. I think it should match the previous code from the diff.

Reply:
prettier removed the brackets around start_page.heading.text

Reply:
It won't work like that without the brackets as `.to` is a method on the return from `expect` not the string that `.text` is.

Reply:
Prettier allows the brackets only if '.to eq(' are on the line above

Reply:
I've just run prettier on my suggestion and it doesn't change it. I think your VS Code prettier plugin may be using different config or something. Try with `bin/lint` in the console. This is this the diff I have locally that prettier is fine with.
```
   def then_i_see_the_start_page
-    expect start_page.heading.text.to eq(
-             "Check your eligibility to apply for qualified teacher status (QTS) in England"
-           )
+    expect(start_page.heading.text).to eq(
+      "Check your eligibility to apply for qualified teacher status (QTS) in England"
+    )
   end
```

Reply:
Could be the space you're leaving after `expect` maybe. 🤷

Reply:
Yeah it was the space